BUILD.bazel

```diff
@@ -2591,7 +2591,6 @@ xnnpack_cc_library(
         "-mavx512vl",
         "-mavx512vnni",
         "-mgfni",
-        "-mamx-int8",
```

Comment:
amx-int8 is required when compiling amx.  if your compile doesnt support amx, the amx needs to be disabled, or the compiler updated.